### DISCUSSION TITLE
question:how can I access other docker services？

### DISCUSSION THREAD

**Discussion by wangweitung** [wg-easy/wg-easy#357] [Q&A]
I run  a docker wg-easy on my Ubuntu server（10.10.10.205）。
After connecting by phone to ubuntu server，I can get access to other devices under my openwrt，like 10.10.10.204。
but I can not access to the Ubuntu server（10.10.10.205），and dockers running on it。
Did I miss some setting？ could u teach me how should I config my settings？Thanks！

**Comment by Matthi-17** [comment]
Hey,
are you running wg-easy on a macvlan?

**Reply by wangweitung** [reply]
> wg-easy

running on wg-easy。

can acess to other container by domain. so problem solved，thanks
